### ISSUE TITLE
Warnings do not show up

### ISSUE THREAD

**Issue by sichvoge** [mulesoft/api-designer#428]
Branch: develop
Milestone: v1
Related to: #403.

Using the following RAML document:

```yaml
#%RAML 0.8
title: Ready

/example:
  get:
    responses: 
      200:
        body: 
          application/json:
            schema: |
              {
                "type":
              }
```

The following warning should pop up "Unrecognized schema".

**Comment by sichvoge** [comment]
Not all the warnings are off. In RAML 1.0 its working.

**Comment by carowright** [comment]
fixed in #422
